extension/encoding/zipkinencodingextension/extension.go

```diff
@@ -76,8 +76,10 @@ func newExtension(config *Config) (*zipkinExtension, error) {
 				unmarshaler: zipkinv1.NewThriftTracesUnmarshaler(),
 			}
 		default:
-			err = fmt.Errorf("unsupported version: %q and protocol: %q", version, protocol)
+			err = fmt.Errorf("protocol: %q, unsupported version: %q", protocol, version)
 		}
+	default:
+		err = fmt.Errorf("version: %q, unsupported protocol: %q", version, protocol)
```

Comment:
Nit: I don't think we should mention a version here because it's applicable to protocol only which is invalid in this case
```suggestion
		err = fmt.Errorf("unsupported protocol: %q", protocol)
```